src/main/java/user/Client.java

```diff
@@ -0,0 +1,21 @@
+package user;
+
+import utils.ClassIdCounter;
+import interfaces.Printable;
+
+public class Client extends User implements Printable {
+    private static final int CLASS_ID = ClassIdCounter.getClassIdCounter();
```

Comment:
the id should not be in the class level

Reply:
Fixed in https://github.com/kolosokvit/ticket-service/pull/2/commits/bb807d65db44ce15c9bf007a1e9cf711391956c4